src/khepri.erl

```diff
@@ -3292,6 +3298,42 @@ transaction(FunOrPath, Args, ReadWrite, Options)
 transaction(StoreId, FunOrPath, Args, ReadWrite, Options) ->
     khepri_machine:transaction(StoreId, FunOrPath, Args, ReadWrite, Options).
 
+-spec handle_async_ret(StoreId, RaEvent) -> ok when
+      StoreId :: khepri:store_id(),
+      RaEvent :: ra_server_proc:ra_event().
```

Comment:
Could you please add a `handle_async_ret/1` variant which uses the default store ID to follow the same pattern as other public functions?